src/framework/preferences/css/stylus/SeparatedPanelPrefsEditor.styl

```diff
@@ -29,7 +29,7 @@
         border-bottom: 1px solid #ccc;
 
         .fl-prefsEditor-buttons {
-            box-shadow: 2px 2px 3px 0px #6f6f6f;
+            box-shadow: none !important; //FLUID-4991: The shadow effect around the hide/show preferences bug fix
```

Comment:
This shouldn't be set to none here. The default style should still have the box-shadow, but you need to make sure that it is being removed when a theme is added.